python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/README.md

```diff
@@ -34,6 +35,7 @@ pip install --pre --upgrade bigdl-llm[all] # install bigdl-llm with 'all' option
 pip install transformers==4.35.0
 pip install accelerate==0.24.1
 pip install einops
+pip install bigdl-llm==2.5.0b20231212
```

Comment:
hi @plusbang , pls take a look. Doesn't `pip install --pre --upgrade bigdl-llm[all]` install nightly version?

Reply:
> hi @plusbang , pls take a look. Doesn't `pip install --pre --upgrade bigdl-llm[all]` install nightly version?

The L34 is exactly `pip install --pre --upgrade bigdl-llm[all] # install bigdl-llm with 'all' option`, why should we need this line? I think we could remove it.